test/controllers/home_controller_test.rb

```diff
@@ -1,8 +1,24 @@
 require 'test_helper'
 
 class HomeControllerTest < ActionController::TestCase
+  setup do
+    @user = users(:one)
+    sign_in @user
+
+    @task = tasks(:one)
+  end
+
   test "should get index" do
     get :index
-    assert_response :success
+    # TODO - Fix?
+  end
+
+  # test "should not see others' tasks" do
+  '''
+    get :dashboard
+
+    assert_equal(@user.tasks.active.count, assigns(:active).count)
+    assert_equal(@user.tasks.archived.count, assigns(:archived).count)
```

Comment:
Tests aren't very helpful when they're commented out!

*P.S. I haven't seen block comments in Ruby in so long, I had to test what these `'''` things did.*